tensorflow/lite/micro/kernels/conv_test.cc

```diff
@@ -325,7 +325,7 @@ TF_LITE_MICRO_TEST(SimpleTestQuantized16x8PerChannelRelu6) {
           filter_quantized, tflite::testing::kBiasShape, bias_values,
           bias_quantized, scales, zero_points, tflite::testing::kOutputShape,
           golden_data, golden_quantized, output_scale, output_zero_point,
-          &conv_params, tflite::Register_CONV_2D(), output_data));
+          &conv_params, tflite::Register_CONV_2D_INT16X8REF(), output_data));
```

Comment:
leave this as Register_CONV_2D (once it calls into EvalInt16x8)

Reply:
done